alexa/skill/briefing.php

```diff
@@ -16,7 +16,11 @@ class Briefing {
 	 * @return array Response for Flash Briefing
 	 */
 	public function briefing_request() {
-		$responses = array();
+		$responses = apply_filters( 'voicewp_override_briefing', array() );
```

Comment:
A WordPress-y name for this filter might include `_pre_`, along the lines of `pre_option_{$option_name}` or `pre_delete_post`. Also, could use [hook docs](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters) if you want.